### PR TITLE
feat(ui) Reworked the Trigger page to show only columns that have data 

### LINKED ISSUE #5546: Rework the Trigger page to show only columns that have data
### Feature description

To reproduce, go to the Administration - Triggers page. You will see that there are too many columns. We should find a creating solution to only display columns that have some data. 

image

### PR DESCRIPTION
**Pull request by MRADUL9936**
### What changes are being made and why?

 made changes in ui/src/components/admin/triggers.vue , because this is the parent component which is responsible for 
fetching the data from store and rendering the table .
I am using a method to filter out the columns which is having no data , and rendering these columns.

Screenshot (253)

### How the changes have been QAed?

  Tested in Different Browsers 

closes #5546

### CODE REVIEW

File: ui/src/components/admin/Triggers.vue [hunk 1]
```diff
@@ -103,65 +99,49 @@
                                     :title="scope.row.namespace + '.' + scope.row.flowId"
                                 />
                             </template>
-                        </el-table-column>
-                        <el-table-column
```
Comment (MilosPaunovic) [inline ui/src/components/admin/Triggers.vue:107]: Where did this column go missing?

File: ui/src/components/admin/Triggers.vue [hunk 2]
```diff
@@ -197,36 +177,6 @@
...
-                                        <play-box />
-                                    </el-tooltip>
-                                    <el-tooltip v-else :content="$t('backfill paused')">
-                                        <pause-box />
-                                    </el-tooltip>
-                                </span>
-                            </template>
-                        </el-table-column>
-
-                        <el-table-column column-key="disable" class-name="row-action">
```
Comment (MilosPaunovic) [inline ui/src/components/admin/Triggers.vue]: Also, this column seem missing too.

**Review by MilosPaunovic — CHANGES_REQUESTED** [review]
Le't try to keep the formatting changes to a minimum, as it will be easier to review PR.

**Comment by Skraye** [comment]
@MRADUL9936 Note that Anna do not ask to remove the column, only to hide them if they are empty, however they are still very important

**Comment by MRADUL9936** [comment]
@MilosPaunovic  I made changes , can u please review it now . , OUTPUT :
Screenshot (270)

**Review by MilosPaunovic — APPROVED** [review]
LGTM! 🚀

**Pull request merged**